manifests/python.pp

```diff
@@ -44,11 +43,19 @@
     require      => Python::Pip['pystan']
   }
 
+  python::pip { 'apache-superset[prophet, postgres]':
+    ensure       => "$version",
```

Comment:
`ensure => $version`

Reply:
Since the value of `$version` is being passed directly to `python::pip::ensure`, it should match the type documented at https://forge.puppet.com/modules/puppet/python/reference#ensure-3.